app/services/proposal_pdf_service.rb

```diff
@@ -63,7 +68,7 @@ def all_proposal_fields
     proposal_organizers
     proposal_locations
     proposal_subjects
-    proposal_bibliography
+    proposal_bibliography unless proposal.no_latex
     user_defined_fields
```

Comment:
bibliography should appear at the bottom of the document, so below user_defined_fields